### PR TITLE
[docs] CLI child commands

### PR DESCRIPTION
**Pull request by sarah-iot**

### CODE REVIEW

File: include/openthread/thread.h [hunk 1]
```diff
@@ -305,7 +305,7 @@ otError otThreadSetJoinerAdvertisement(otInstance *   aInstance,
 #define OT_JOINER_ADVDATA_MAX_LENGTH 64 ///< Maximum AdvData Length of Joiner Advertisement
 
 /**
- * Get the Thread Child Timeout used when operating in the Child role.
+ * Gets the Thread Child Timeout seconds used when operating in the Child role.
```
Comment (jwhui) [inline include/openthread/thread.h:308]: nit: we typically mention units as follows.
```suggestion
 * Gets the Thread Child Timeout (in seconds) used when operating in the Child role.
```

File: include/openthread/thread.h [hunk 2]
```diff
@@ -317,7 +317,7 @@ otError otThreadSetJoinerAdvertisement(otInstance *   aInstance,
 uint32_t otThreadGetChildTimeout(otInstance *aInstance);
 
 /**
- * Set the Thread Child Timeout used when operating in the Child role.
+ * Sets the Thread Child Timeout seconds used when operating in the Child role.
```
Comment (jwhui) [inline include/openthread/thread.h:320]:
```suggestion
 * Sets the Thread Child Timeout (in seconds) used when operating in the Child role.
```

**Review by abtink — APPROVED** [review]
Thanks. LGTM. 👍

**Comment by jwhui** [comment]
@sarah-iot , you'll need to rebase and resolve conflicts.

**Comment by sarah-iot** [comment]
OK, rebased & updated; fyi @Vyrastas

**Review by Vyrastas — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**